addons/binding/org.openhab.binding.vitotronic/ESH-INF/thing/thing-types.xml

```diff
@@ -15,12 +15,41 @@
 
         <channels>
             <channel id="systemtime" typeId="systemtime"/>
-            <channel id="outside_temp" typeId="outside_temp"/>
-            <channel id="boiler_temp" typeId="boiler_temp"/>   
-            <channel id="flowuprating" typeId="flowuprating"/>   
+            <channel id="outside_temp" typeId="temperature_ro">

```

Comment:
Sorry, you will need to rework your channel types - they must not be reused for different kind of functionality.
The old definition was correct in this respect - there is more or less one channel type per channel and no re-use.

See [the documentation](https://www.eclipse.org/smarthome/documentation/development/bindings/thing-definition.html#channels) about this:

> The granularity of channel types should be on its semantic level, i.e. very fine-grained: If a Thing measures two temperature values, one for indoor and one for outdoor, this should be modelled as two different channel types. Overriding labels of a channel type must only be done if the very same functionality is offered multiple times, e.g. having an actuator with 5 relais, which each is a simple “switch”, but you want to individually name the channels (1-5).